pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/ExcelParticipantExporter.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.dsm.model.elastic.export.tabular;
+
+import javax.servlet.ServletOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import com.google.common.net.MediaType;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.streaming.SXSSFSheet;
+import org.apache.poi.xssf.streaming.SXSSFWorkbook;
+import spark.Response;
+
+public class ExcelParticipantExporter extends TabularParticipantExporter {
+    private final SXSSFWorkbook workbook = new SXSSFWorkbook(200);
+
+    private final SXSSFSheet sheet;
+    private static final String SHEET_NAME = "Participant List";
+
+
+    public ExcelParticipantExporter(List<ModuleExportConfig> moduleConfigs,
+                                    List<Map<String, String>> participantValueMaps, String fileFormat) {
+        super(moduleConfigs, participantValueMaps, fileFormat);
+        this.sheet = workbook.createSheet(SHEET_NAME);
+        sheet.trackAllColumnsForAutoSizing();
+    }
+
+    public void export(Response response) throws IOException {
+        setResponseHeaders(response);
+
+        List<String> headerValues = getHeaderRow();
+        List<String> subHeaderValues = getSubHeaderRow();
+
+        writeRowToSheet(headerValues, 0);
+        writeRowToSheet(subHeaderValues, 1);
+
+        IntStream.range(0, participantValueMaps.size()).forEach(i -> {
+            Map<String, String> valueMap = participantValueMaps.get(i);
+            List<String> rowValues = getRowValues(valueMap, headerValues);
+            writeRowToSheet(rowValues, i + 2);
+        });
+
+        try (ServletOutputStream servOut = response.raw().getOutputStream()) {
+            workbook.write(servOut);
+        }
+        workbook.dispose();
+        workbook.close();
```

Comment:
having workbook inside `try-with-resource` like `try (ServletOutputStream servOut = response.raw().getOutputStream(); workbook)` would not automatically close the workbook?

Reply:
good question, I don't think so, since the workbook is instantiated outside of the block.  I'll double-check

Reply:
yup, we need these outside the loop.  The alternative would be to create the entire sheet inside the `try`, but I *think* its more efficient to not hold the output stream open for that whole time.  We can revisit if we need to start piecemeal streaming exports